ouroboros-consensus-diffusion/changelog.d/chaindb-args-javier.md

```diff
@@ -0,0 +1,8 @@
+### Breaking
+
+- Removed `mkChainDbArgs`.
+- New `llrnMkHasFS` field in `LowLevelRunNodeArgs`
+
+### Non-Breaking
+
+- Adapted to `ChainDbArgs` rework in `ouroboros-consensus`.
```

Comment:
`O.C.Node` re-exports `ChainDbArgs (..)`, so one could argue that there is more breaking stuff going on

Reply:
Added